cscs-checks/microbenchmarks/osu/osu_tests.py

```diff
@@ -32,14 +32,18 @@ def __init__(self, variant):
             'daint:gpu': {
                 'latency': (20.73, None, 2.0, 'us')
             },
+            '*': {
+                'latency': (0, None, None, 'us')
+            },
         }
         self.num_tasks_per_node = 1
         self.num_gpus_per_node  = 1
         if self.current_system.name == 'dom':
             self.num_tasks = 6
-
-        if self.current_system.name == 'daint':
+        elif self.current_system.name == 'daint':
             self.num_tasks = 16
+        else:
+            self.num_tasks = 2
```

Comment:
After second thoughts, I believe that we should not do that. The test is meant for X nodes; we should not assume less for an unknown system. The `sbatch` will fail if the request cannot be satisfied. This test has another problem though; it runs different configurations for Daint/Dom; we should do it as for the AllReduce with two variants, `small` and `large`. This is different issue.